ffi/android/Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM rustlang/rust:nightly-slim as builder
+
+# mkdir required to install openjdk on slim images
+RUN mkdir -p /usr/share/man/man1 && \
+    apt-get update && \
+    apt-get install -y \
+        cmake g++ git libpcre3-dev make openjdk-11-jdk unzip wget && \
+    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+WORKDIR /tmp
+
+RUN wget https://freefr.dl.sourceforge.net/project/swig/swig/swig-4.0.1/swig-4.0.1.tar.gz && \
+    tar xf swig-*.tar.gz && \
+    cd swig-* && \
+    ./configure && make -j12 && \
+    make install && \
+    rm -rf /tmp/*
+
+ENV ANDROID_SDK_ROOT=/root/sdk
+ENV NDK_VERSION=20.1.5948944
+ENV NDK_HOME=$ANDROID_SDK_ROOT/ndk/$NDK_VERSION
```

Comment:
Maybe better make them `ARG`s?

Reply:
`NDK_VERSION` can be an `ARG`, but `ANDROID_SDK_ROOT` and `NDK_HOME` are more useful as env variables, since one could launch the image to debug the bindings from inside a container, and those variables are needed for build

Reply:
Good, can we do that?

Reply:
yep, done